pkgs/by-name/mo/monophony/package.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, fetchFromGitLab
+, python3Packages
+, wrapGAppsHook4
+, gst_all_1
+, gobject-introspection
+, gtk4
+, yt-dlp
+, libadwaita
+, libsoup_3
+, glib-networking
+,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "monophony";
+  version = "2.2.2";
```

Comment:
```suggestion
  version = "2.3.0";
```

Reply:
Oh lol, it must have gotten a release since i started, will do

Reply:
Fixed